home.html

```diff
@@ -223,5 +223,93 @@ <h1 class="aboutme-heading">About Me</h1>
         </ul>
       </div>
     </section>
+    <section class="form-section">
+      <h3 class="form-description">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+      <form
+        class="portfolio-form"
+        action="https://formspree.io/f/mdobanyg"
+        method="post"
+      >
+        <ul>
+          <li>
+            <input
+              class="form-input"
+              type="text"
+              id="name"
+              name="name"
+              placeholder="Full name"
+              maxlength="30"
+              required
+            />
+          </li>
+          <li>
+            <input
+              class="form-input"
+              type="email"
+              id="email"
+              name="email"
+              placeholder="Email address"
+              required
+            />
+          </li>
+          <li>
+            <textarea
+              class="form-message"
+              id="message"
+              name="message"
+              placeholder="Enter text here"
+              maxlength="500"
+              required
+            ></textarea>
+          </li>
+        </ul>
+        <input
+          class="btn btn-solid-short form-btn"
+          type="submit"
+          value="Get in touch"
+          name="submit"
+        />
+      </form>
+
+      <footer class="footer">
+        <hr class="thin-line" />
+        <div class="button-group-1 footer-buttons">
+          <a href="#"
+            ><img
+              src="/assets/images/icon-github-dark.svg"
+              alt="Github icon"
+              class="github-icon"
+          /></a>
+          <a href="#">
+            <img
+              src="/assets/images/icon-linkden-dark.svg"
+              alt="linkedin icon"
+              class="linkedin-icon"
+          /></a>
+          <a href="#">
+            <img
+              src="/assets/images/icon-angelist-dark.svg"
+              alt="Angelist icon"
+              class="angelist-icon"
+          /></a>
+          <a href="#"
+            ><img
+              src="/assets/images/icon-twitter-dark.svg"
+              alt="Twitter icon"
+              class="twitter-icon"
+          /></a>
+          <a href="#"
+            ><img
+              src="/assets/images/icon-medium-dark.svg"
+              alt="Medium icon"
+              class="medium-icon"
```

Comment:
Please ensure you stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding).

## Your design:
![your-design](https://user-images.githubusercontent.com/10332499/149036432-20403cd9-7cf6-401c-aee2-c7b0afd817eb.PNG)

## Figma design:
![figma](https://user-images.githubusercontent.com/10332499/149036460-50cf6b4b-3f2b-4ace-9b9a-f10a929af4f6.PNG)

Reply:
Hi @micheaol 

From your suggestion, I can see that the "textarea" background is different from the Figma reference design file. But I am unable to figure out the other details that require change. Would you please point me to which portion I need to apply the change?

Reply:
@mengiefen 
Please also ensure that the social icons are visible and the contents are properly aligned just like Figma's design.